index.bs

```diff
@@ -5784,13 +5780,12 @@ To <dfn>get the initiator</dfn> given |request|:
   TODO: Chrome includes the current parser position as column number / line
   number for parser-inserted resources.
 
-1. Return a [=/map=] matching the <code>network.Initiator</code> production, with
-   the <code>type</code> field set to |type|, the <code>columnNumber</code>
-   field set to |column number| if it's not null, or omitted otherwise, the
-   <code>lineNumber</code> field set to |line number| if it's not null, or
-   omitted otherwise, the <code>stackTrace</code> field set to |stack trace| if
-   it's not null, or omitted otherwise, and the <code>request</code> field set
-   to |request id|.
+1. Return a [=/map=] matching the <code>network.Initiator</code> production, the
+   <code>columnNumber</code> field set to |column number| if it's not null, or
+   omitted otherwise, the <code>lineNumber</code> field set to |line number| if
+   it's not null, or omitted otherwise, the <code>stackTrace</code> field set to
+   |stack trace| if it's not null, or omitted otherwise, and the
+   <code>request</code> field set to |request id|.
```

Comment:
I know this was merged already but based on the definition here, `request` is not actually optional and can never be missing, which means Initiator can never be empty and should not be optional in the BeforeRequestSentParameters ? Or should we remove request from the map?